src/Common/DocumentUtility.ts

```diff
@@ -2,7 +2,7 @@ import { DefaultAccountExperienceType } from "../DefaultAccountExperienceType";
 import { userContext } from "../UserContext";
 
 export const getEntityName = (): string => {
-  if (userContext.defaultExperience === DefaultAccountExperienceType.MongoDB) {
+  if (userContext.apiType === DefaultAccountExperienceType.MongoDB) {
```

Comment:
This right here should be throwing a compiler error. It does in vscode for me! But not when running tsc or in CI.
We should wait on this PR until we have unraveled this mystery

Reply:
I have changed value of DefaultAccountExperienceType. as we have discussed mapping between DefaultAccountExperienceType and apiType value.


`export enum DefaultAccountExperienceType {
  DocumentDB = "SQL",
  Graph = "Gremlin",
  MongoDB = "Mongo",
  Table = "Tables",
  Cassandra = "Cassandra",
  ApiForMongoDB = "Mongo",
}
`

type ApiType = "SQL" | "Mongo" | "Gremlin" | "Tables" | "Cassandra";

If I don't change the value of DefaultAccountExperienceType, so It will give following error and also give typescript error.

This condition will always return 'false' since the types 'ApiType' and 'DefaultAccountExperienceType.MongoDB' have no overlap.ts(2367)

Reply:
I actually want that behavior. Remove `DefaultAccountExperienceType` completely and change all comparisons to use the literal strings:

userContext.apiType === "Mongo"

Reply:
Resolve by removing DefaultAccountExperienceType